snap/local/update-device-configuration.sh

```diff
@@ -2,4 +2,15 @@
 
 CONFIGURATION_FILE_PATH="$SNAP_COMMON/configuration/device.yaml"
 
-$SNAP/bin/cos-registration-agent --shared-data-path $SNAP_COMMON/rob-cos-shared-data --grafana-dashboards $SNAP_COMMON/configuration/grafana_dashboards --foxglove-studio-dashboards $SNAP_COMMON/configuration/foxglove_layouts update -c $CONFIGURATION_FILE_PATH
+# Check if grafan_dashboards directory exists in configuration
+if [ -d "${SNAP_COMMON}/configuration/grafana_dashboards" ]; then
+    REGISTRATION_CMD_ARGS="${REGISTRATION_CMD_ARGS} --grafana-dashboards ${SNAP_COMMON}/configuration/grafana_dashboards"
```

Comment:
Please define this `REGISTRATION_CMD_ARGS` variable with an empty string at the beginning of the script.
If no folders are found, the variable is unbounded.

Reply:
good catch, added in 7a58855e865bce725abda751ce2431631c6425c1